service/src/main/java/bio/terra/pipelines/service/PipelinesService.java

```diff
@@ -91,6 +91,7 @@ public void validateUserProvidedInputs(
     checkForExtraInputs(userProvidedInputDefinitions, inputsMap);
 
     if (!errorMessages.isEmpty()) {
+      errorMessages.sort(String::compareToIgnoreCase); // sort to help with testing
```

Comment:
hmm we shouldnt be doing this if its just for testing.  In the test we should just assert what we can.